### PR TITLE
Reinstate link to Haskell Platform repository

### PR DESCRIPTION
**Pull request by tomjaguarpaw**
... and add link to ghcup subsite

Fixes https://github.com/haskell-infra/www.haskell.org/issues/77

### CODE REVIEW

**Comment by tomjaguarpaw** [comment]
(Link previously removed due to misunderstanding)

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ Not all subsites of www.haskell.org are built from this repository.
 Some of the others are
 
 * www.haskell.org/cabal (built from [cabal-website](https://github.com/haskell/cabal-website))
+* www.haskell.org/platform (built from [haskell-platform](https://github.com/haskell/haskell-platform/tree/master/website))
```
Comment (hasufell) [inline README.md:11]: I guess https://www.haskell.org/ghcup/ is missing too... the repo is here: https://gitlab.haskell.org/haskell/ghcup-hs/-/tree/master/www

Comment (tomjaguarpaw) [reply README.md:11]: Thanks! I've added it to this PR.

**Review by TikhonJelvis — APPROVED** [review]
Looks good!

**Pull request merged**